test/controllers/redis_log_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class RedisLogControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Intend to have this here? Assume it'll fail b/c auth.

Reply:
Nah, added by rails generator